src/sql/workbench/api/node/extHostConnectionManagement.ts

```diff
@@ -21,7 +21,9 @@ export class ExtHostConnectionManagement extends ExtHostConnectionManagementShap
 	public $getCurrentConnection(): Thenable<azdata.connection.ConnectionProfile> {
 		let connection: any = this._proxy.$getCurrentConnection();
 		connection.then((conn) => {
-			conn.providerId = conn.providerName;
+			if (conn) {
```

Comment:
is it worth logging something to the console when conn is undefined?  If the logger isn't easily accessible at this place in the code it's probably fine as is.